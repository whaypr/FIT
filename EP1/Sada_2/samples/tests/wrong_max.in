248
1 365
2 277
3 264
4 534
5 637
6 461
7 408
8 226
9 130
10 117
11 399
12 44
13 648
14 633
15 271
16 583
17 578
18 598
19 711
20 687
21 263
22 403
23 289
24 241
25 466
26 571
27 174
28 245
29 468
30 514
31 297
32 672
33 150
34 86
35 253
36 252
37 395
38 637
39 197
40 613
41 240
42 717
43 550
44 719
45 377
46 427
47 490
48 506
49 607
50 732
51 101
52 719
53 246
54 647
55 715
56 174
57 285
58 637
59 20
60 151
61 505
62 307
63 104
64 380
65 22
66 209
67 364
68 252
69 630
70 20
71 212
72 623
73 44
74 296
75 599
76 526
77 264
78 638
79 373
80 354
81 553
82 658
83 328
84 641
85 468
86 49
87 277
88 194
89 411
90 334
91 225
92 245
93 608
94 137
95 247
96 610
97 672
98 502
99 684
100 229
101 401
102 205
103 556
104 327
105 60
106 418
107 495
108 263
109 667
110 481
111 465
112 194
113 620
114 377
115 180
116 629
117 246
118 544
119 7
120 224
121 120
122 631
123 80
124 139
125 385
126 307
127 410
128 253
129 575
130 477
131 243
132 338
133 602
134 490
135 20
136 404
137 48
138 700
139 189
140 356
141 105
142 257
143 15
144 257
145 29
146 253
147 707
148 187
149 457
150 692
151 451
152 296
153 323
154 198
155 548
156 89
157 644
158 178
159 361
160 392
161 46
162 213
163 479
164 265
165 477
166 29
167 707
168 578
169 507
170 498
171 468
172 534
173 674
174 457
175 485
176 9
177 102
178 120
179 59
180 267
181 47
182 182
183 95
184 516
185 675
186 461
187 540
188 708
189 48
190 627
191 235
192 398
193 538
194 124
195 474
196 606
197 407
198 552
199 612
200 408
201 173
202 419
203 74
204 316
205 561
206 258
207 38
208 552
209 208
210 483
211 425
212 212
213 40
214 346
215 153
216 684
217 49
218 229
219 187
220 180
221 454
222 405
223 526
224 19
225 80
226 103
227 197
228 675
229 503
230 274
231 678
232 411
233 268
234 566
235 107
236 276
237 316
238 445
239 541
240 478
241 626
242 727
243 356
244 403
245 625
246 683
247 572
248 562
10
gt min 166
gt avg 75
gt avg 139
gt max 111
lt min 58
gt max 70
lt min 253
lt avg 150
lt avg 200
gt avg 189
