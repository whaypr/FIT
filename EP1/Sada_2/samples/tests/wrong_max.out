238
123
122
9
8
11
8
126
126
121
